src/scribe_data/cli/list.py

```diff
@@ -0,0 +1,153 @@
+import json
```

Comment:
We need the license note that the top of this file, but I'm adding it to my local version of the PR now :)